projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx

```diff
@@ -52,7 +54,7 @@ export const AiAssistantPopover = ( {
 				} }
 			/>
 
-			<div style={ { width } }>
+			<div style={ { width: '100%' } }>
```

Comment:
why not use simply CSS?

Reply:
I just copy and paste from current code: https://github.com/Automattic/jetpack/blob/1b1b2f0c619c313f9f6ba3f9b50010bd684da191/projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-toolbar-button/index.tsx#L73 ... Let me update it.

Reply:
well... that's part of the code to clean up. Thanks